src/components/Counter.js

```diff
@@ -258,10 +274,6 @@ export default class Counter extends React.Component {
         </div>
       )
     })
-    return (
-      <div className={this.getCSSClasses()}>
-        {segments}
-      </div>
-    )
+    return <div className={this.getCSSClasses()}>{segments}</div>
```

Comment:
brackets are requried in `return(<jsx ... />)`

Reply:
No, brackets are only required if the JSX is multiline.